Ginger/GingerCoreNET/Reports/BusinessFlowReport.cs

```diff
@@ -56,6 +56,7 @@ public static partial class Fields
             public static string Elapsed = "Elapsed";
             public static string RunStatus = "RunStatus";
             public static string NumberOfActivities = "NumberOfActivities";
+            public static string ExecutionRate = "ExecutionRate";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'ExecutionRate' or make it 'const' or 'readonly'.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)